extensions/binding/org.eclipse.smarthome.binding.hue.test/src/test/groovy/org/eclipse/smarthome/binding/hue/test/HueLightHandlerOSGiTest.groovy

```diff
@@ -428,7 +426,7 @@ class HueLightHandlerOSGiTest extends OSGiTest {
             assertBridgeOnline(hueLightHandler.getBridge())
             enableHueLightChannels(hueLight)
             //Run the initialize method in case of PAR16 50 TW bulb testcase
-            if(currentState.modelId == 'PAR16 50 TW') {
+            if(currentState.modelId.equals(OSRAM_PAR16_MODELID)) {
```

Comment:
Can you remove this condition and always call the initialize() method? I don't think there is a reason to do it only for the OSRAM light.